src/WebKit/WebKit.cs

```diff
@@ -1,7 +1,16 @@
+using System;
+using System.Runtime.Versioning;
 using Foundation;
 
 namespace WebKit {
-	
+
+#if NET
+	[SupportedOSPlatform ("macos")]
+	[UnsupportedOSPlatform ("macos10.14")]
+#if MONOMAC
+	[Obsolete ("Starting with macos10.14 no longer supported.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+#endif
```

Comment:
Wouldn't the generated part of this partial class have these attributes?

Reply:
Is this a question about this specific instance or the PR in general?

Right now, the generated code only has:

```
	[Register("WebFrame", true)]
	[UnsupportedOSPlatform ("macos10.14")]
	public unsafe partial class WebFrame : NSObject {
```

but it should match this when I finish the generator PR. 

Part of my question on Teams (that I referenced in the description) was if we should "duplicate" attributes on partial class definitions.

Reply:
Ok, that changes things. I'll come up with a plan after lunch, likely just reverting commits by hand this one time.

Reply:
I have a local patch that removes all changes on public partial classes building locally.

Thanks [patch iron](https://github.com/chamons/patch-iron/blob/master/Conversion.cs) and past-me for writing it to make this an hour-ish job instead of many hours.